base/bucket.go

```diff
@@ -188,12 +200,32 @@ func (spec *BucketSpec) GetGoCBConnString(params *GoCBConnStringParams) (string,
 	asValues := url.Values(connSpec.Options)
 
 	// Add kv_pool_size as used in both GoCB versions
-	poolSizeFromConnStr := asValues.Get("kv_pool_size")
-	if poolSizeFromConnStr == "" {
+	fromConnStr := asValues.Get("kv_pool_size")
+	if fromConnStr == "" {
 		asValues.Set("kv_pool_size", strconv.Itoa(params.KVPoolSize))
 		spec.KvPoolSize = params.KVPoolSize
 	} else {
-		spec.KvPoolSize, _ = strconv.Atoi(poolSizeFromConnStr)
+		spec.KvPoolSize, _ = strconv.Atoi(fromConnStr)
+	}
+
+	fromConnStr = asValues.Get("kv_buffer_size")
```

Comment:
Reusing the fromConnStr variable isn't helpful here I don't think - it would be clearer if the previous naming convention was used (poolSizeFromConnStr, kvBufferSizeFromConnStr, etc)